app/uk/gov/hmrc/ctcguaranteebalanceeisstub/models/responses/BalanceResponse.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ */
+
+package uk.gov.hmrc.ctcguaranteebalanceeisstub.models.responses
+
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json.Format
+import play.api.libs.json.Json
+import play.api.libs.json.__
+import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.AccessCode
+import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.GuaranteeReferenceNumber
+
+import scala.util.Random
+
+case class BalanceResponse(guaranteeReferenceNumber: GuaranteeReferenceNumber, remainingBalance: Balance)
+
+case class Balance(value: Int) extends AnyVal
+
+object Balance {
+  implicit val balanceFormat = Json.valueFormat[Balance]
+
+  def generateBalance(): Balance = Balance(Random.nextInt(10000))
```

Comment:
I'm not too sure how I feel about this from a testing point of view -- the stub is mostly designed for us to be able to test with. I think it's probably worth asking Kiran how he wants to be able to test things -- do we have a constant balance, or something calculated (and predictable) based on the GRN?

A random number is going to be hard to test. Same with the Access Code.

Reply:
Yes makes sense Dan, will update to a constant value.